hack/godep-restore.sh

```diff
@@ -22,8 +22,15 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 source "${KUBE_ROOT}/hack/lib/util.sh"
 
-kube::util::ensure_godep_version v79
+kube::log::status "Restoring kubernetes godeps"
 
-kube::log::status "Starting to download all kubernetes godeps. This takes a while"
-GOPATH=${GOPATH}:${KUBE_ROOT}/staging godep restore "$@"
-kube::log::status "Download finished"
+if kube::util::godep_restored >/dev/null 2>&1; then
+    kube::log::status "Dependencies appear to be current - skipping download"
+    exit 0
+fi
+
+kube::util::ensure_godep_version
+
+kube::log::status "Downloading dependencies - this might take a while"
+GOPATH="${GOPATH}:$(pwd)/staging" godep restore "$@"
```

Comment:
Is there a reason you're using pwd over KUBE_ROOT?

Reply:
From an earlier rev.  Will fix